src/connectors/articleService.ts

```diff
@@ -624,24 +624,37 @@ export class ArticleService extends BaseService {
    * Archive article
    */
   archive = async (id: string) => {
-    // update search
-    try {
-      await this.es.client.update({
-        index: this.table,
-        id,
-        body: {
-          doc: { state: ARTICLE_STATE.archived },
-        },
+    const atomService = new AtomService()
+    const targetArticle = await atomService.findFirst({
+      table: 'article',
+      where: { id },
+    })
+    const articles = await atomService.findMany({
+      table: 'article',
+      where: { draftId: targetArticle.draftId },
+    })
+
+    // update db
+    for (const article of articles) {
+      await this.baseUpdate(article.id, {
+        state: ARTICLE_STATE.archived,
+        sticky: false,
+        updatedAt: new Date(),
```

Comment:
in many cases, we have changed this to `updatedAt: knex.fn.now()` which has better timestamp precision, better in later analysis

Reply:
and this findMany & for-loop look like might be just one call of `await atomService.updateMany` ?

Reply:
I will make another PR to `develop`